resources/language/resource.language.en_gb/strings.po

```diff
@@ -1,793 +1,825 @@
+msgstr ""
+
+msgctxt "#30157"
+msgid "The InputStream settings panel will now be opened, so you have to change this option \"Override HDCP status\" to ON"
+msgstr ""
+
+msgctxt "#30158"
+msgid "Show configuration screen at addon startup"
+msgstr ""
+
+msgctxt "#30159"
+msgid "View for main menu"
+msgstr ""
+
+msgctxt "#30160"
+msgid "View for search"
+msgstr ""
+
+msgctxt "#30161"
+msgid "View for exported"
+msgstr ""
+
+msgctxt "#30162"
+msgid "Last used"
+msgstr ""
+
+msgctxt "#30163"
+msgid "Audio description"
+msgstr ""
+
+msgctxt "#30164"
+msgid "Browse contents with audio description."
+msgstr ""
+
+msgctxt "#30165"
+msgid "View for my list"
+msgstr ""
+
+msgctxt "#30166"
+msgid "Main menu items"
+msgstr ""
+
+msgctxt "#30167"
+msgid "My list"
+msgstr ""
+
+msgctxt "#30168"
+msgid "Continue watching"
+msgstr ""
+
+msgctxt "#30169"
+msgid "Top picks"
+msgstr ""
+
+msgctxt "#30170"
+msgid "New releases"
+msgstr ""
+
+msgctxt "#30171"
+msgid "Trending now"
+msgstr ""
+
+msgctxt "#30172"
+msgid "Popular on Netflix"
+msgstr ""
+
+msgctxt "#30173"
+msgid "NETFLIX ORIGINALS"
+msgstr ""
+
+msgctxt "#30174"
+msgid "Tv Show genres"
+msgstr ""
+
+msgctxt "#30175"
+msgid "Movie genres"
+msgstr ""
+
+msgctxt "#30176"
+msgid "Enable STRM resume workaround (only library)"
+msgstr ""
+
+msgctxt "#30177"
+msgid "Ask to resume the video"
+msgstr ""
+
+msgctxt "#30178"
+msgid "Open UpNext Addon settings"
+msgstr ""
+
+msgctxt "#30179"
+msgid "Show the trailers"
+msgstr ""
+
+msgctxt "#30180"
+msgid "No trailers available"
+msgstr ""
+
+msgctxt "#30181"
+msgid "Disable subtitles if there are no forced subtitles streams (Works only if Kodi Player is set to \"Only forced\" subtitles)"
+msgstr ""
+
+msgctxt "#30182"
+msgid "Export NFO files"
+msgstr ""
+
+msgctxt "#30183"
+msgid "Do you want to export NFO files for the {}?"
+msgstr ""
+
+msgctxt "#30184"
+msgid "NFO Files"
+msgstr ""
+
+msgctxt "#30185"
+msgid "Enable NFO files export"
+msgstr ""
+
+msgctxt "#30186"
+msgid "For Movies"
+msgstr ""
+
+msgctxt "#30187"
+msgid "For TV Show"
+msgstr ""
+
+msgctxt "#30188"
+msgid "Ask"
+msgstr ""
+
+msgctxt "#30189"
+msgid "movie"
+msgstr ""
+
+msgctxt "#30190"
+msgid "TV show"
+msgstr ""
+
+msgctxt "#30191"
+msgid "Do you want to export NFO files for the {} added to 'My List'?"
+msgstr ""
+
+msgctxt "#30192"
+msgid "\r\nThese files are used by the provider information scrapers to integrate movies and tv shows info, useful with bonus episodes or director's cut"
+msgstr ""
+
+msgctxt "#30193"
+msgid "Include all information in NFO files (use only with 'Local Information' scraper)"
+msgstr ""
+
+msgctxt "#30194"
+msgid "Stream Type"
+msgstr ""
+
+msgctxt "#30195"
+msgid "Video Strem Type (Up to...)"
+msgstr ""
+
+msgctxt "#30196"
+msgid "SD"
+msgstr ""
+
+msgctxt "#30197"
+msgid "640"
+msgstr ""
+
+msgctxt "#30198"
+msgid "720p"
+msgstr ""
+
+msgctxt "#30199"
+msgid "1080p"

```

Comment:
HD 1080p ?

Reply:
I think we need to be standard, so if we use SD/HD res_h"p" its oks,